### DISCUSSION TITLE
Request: Export all pages to markdown

### DISCUSSION THREAD

**Discussion by markusstraub** [growilabs/growi#7718] [Ideas / Requests] [answered]
I am pondering the switch of a wiki to growi but want to avoid lock-in of the data, i.e. I only found https://docs.growi.org/en/admin-guide/management-cookbook/export.html for exporting data, but this is a custom format only usable with growi.

Is there a possibility to export (at least the current version of) all pages to markdown files? I imagine something like a .zip with one .md file for each page.

**Comment by yuki-takei** [comment]
@markusstraub 
Hi. I know users want it, and we considered to start to implement it several weeks ago, but there is no prospect of release yet...

**Comment by sevenjay** [comment]
We also need this feature.
Please implement to export (at least the current version of) all pages to markdown files.
We have an AI bot, and prepare to use Growi as our knowledge base.
There are hundreds pages in our Growi and Growi.cloud. (Yes, we both deployed for different departments)
The AI bot can be imported markdown files, just ask, it will find the answer.
Thanks for your help.

**Comment by yuki-takei** [comment, marked as answer]
This feature has been implemented since [v7.3.0](https://github.com/growilabs/growi/releases/tag/v7.3.0)! Pages can now be exported in Markdown format:

- **Single page**: Use the page context menu → "Export as Markdown"
- **Bulk export**: Use the Bulk Export modal to export multiple pages in Markdown (`.md`) or PDF format

Thank you for your suggestion!
